UnitTests/ReportTests.swift

```diff
@@ -0,0 +1,108 @@
+//
+//  ReportTests.swift
+//  UnitTests
+//
+//  Created by Martin Dutra on 14/7/22.
+//  Copyright © 2022 Verse Communications Inc. All rights reserved.
+//
+
+import XCTest
+
+class ReportTests: XCTestCase {
+
+    var tmpURL = URL(fileURLWithPath: "unset")
+    var db = ViewDatabase()
+    let testAuthor: Identity = DatabaseFixture.exampleFeed.identities[0]
+
+    override func setUpWithError() throws {
+        try super.setUpWithError()
+        db.close()
+
+        // get random location for the new
+        tmpURL = URL(fileURLWithPath: NSTemporaryDirectory().appending("/viewDBtest-feedFill2"))
+
+        do {
+            try FileManager.default.removeItem(at: tmpURL)
+        } catch {
+            // ignore - most likely not exists
+        }
+
+        try FileManager.default.createDirectory(at: tmpURL, withIntermediateDirectories: true)
+
+        // open DB
+        let dbPath = tmpURL.absoluteString.replacingOccurrences(of: "file://", with: "")
+        try db.open(path: dbPath, user: testAuthor, maxAge: -60 * 60 * 24 * 30 * 48)
```

Comment:
Might want to make the max age MAX_INT or something because I see there are some hard coded dates below so the tests could maybe start to fail in 6 months.